src/ec/packages/ec-component-timeline/timeline.html.twig

```diff
@@ -0,0 +1,106 @@
+{% set _items = items|default([]) %}
+{% set _icon_path = icon_path|default('') %}
+{% set _toggle_collapsed = toggleCollapsed|default('Show %d more items') %}
+{% set _toggle_expanded = toggleExpanded|default('Hide %d items') %}
+
+
+{% set display_amount = items|length %}
+{% set hide_amount = 0 %}
+{% set bottom_amount = 0 %}
+
+{% if items|length >= 13 %}
+  {% set bottom_amount = 2 %}
+  {% set countbase = items|length - bottom_amount %}
+  {% set display_amount = 7 %}
+  {% if (countbase / 2) < 7 %}
+    {% set display_amount = countbase / 2 %}
+  {% endif %}
+  {% set hide_amount = countbase - display_amount %}
+{% endif %}
+
+{# Internal logic - Process properties #}
+
+{% if extra_classes is defined and extra_classes is not empty %}
+  {% set _css_class = _css_class ~ ' ' ~ extra_classes %}
+{% endif %}
+
+{% if extra_attributes is defined and extra_attributes is not empty and extra_attributes is iterable %}
+  {% for attr in extra_attributes %}
+    {% set _extra_attributes = _extra_attributes ~ ' ' ~ attr.name ~ '="' ~ attr.value ~ '"' %}
+  {% endfor %}
+{% endif %}
+
+{# Print the result #}
+
+<ol class="{{ _css_class }}"{{ _extra_attributes|raw }}
+    data-ecl-timeline="true">
+  {% if _items is not empty %}
+    {% for _item in _items %}
+      {# Display the show more button if required.#}
+      {% if hide_amount > 0 and loop.revindex == 2 %}
+        <li class="ecl-timeline2__item ecl-timeline2__item--toggle">
+        <button data-ecl-timeline-button="true"
+                data-ecl-label-collapsed="{{ toggleCollapsed|replace({'%d': countbase - display_amount}) }}"
+                data-ecl-label-expanded="{{ toggleExpanded|replace({'%d': countbase - display_amount}) }}"
+                type="button"
+                class="ecl-timeline2__toggle ecl-button ecl-button--secondary"><span
+            class="ecl-button__container"><span
+              class="ecl-button__label"
+              data-ecl-label="true">{{ toggleCollapsed|replace({'%d': countbase - display_amount}) }}</span>
+            {% include '../ec-component-icon/icon.html.twig' with {
+              icon: {
+                path: _icon_path,
+                type: "ui",
+                name: "corner-arrow",
+                size: "xs"
+              },
+              extra_classes: 'ecl-icon--rotate-180 ecl-button__icon ecl-button__icon--after',
+            } only %}
+          </span>
+        </button>
+      {% endif %}
+      {# Deside on per item if it should display. #}
+      {% set extra_class = (loop.index > display_amount and loop.revindex > 2) ? ' ecl-timeline2__item--collapsed' : '' %}
+      <li class="ecl-timeline2__item{{ extra_class }}" id="{{ _item.id }}">
+        <div class="ecl-timeline2__label">{{ _item.label }}</div>
+        <div class="ecl-timeline2__title">{{ _item.title }}</div>
+        <div class="ecl-timeline2__content">
+          {%- set _content = _item.content|default('') %}
+          {%- block content _content -%}
```

Comment:
I'm not sure this block is usable. How would a parent template embed the timeline and use the `content` block?

Reply:
Changed this to {{_content}}